keyboards/dtd/ldmlKeyboard3.dtd

```diff
@@ -10,7 +10,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ELEMENT keyboard3 ( import*, locales?, version?, info, settings?, displays?, keys?, flicks?, forms?, layers*, variables?, transforms*, special* ) >
 <!ATTLIST keyboard3 locale CDATA #REQUIRED >
     <!--@MATCH:validity/bcp47-wellformed-->
-<!ATTLIST keyboard3 conformsTo (45 | 46) #REQUIRED >
+<!ATTLIST keyboard3 conformsTo (45 | 46 | 46.1) #REQUIRED >
```

Comment:
@srl295 This is causing a CI test failure, with error as shown below. If I run GenerateDtd as suggested it replaces the "46.1" with "1".. Is there some problem with decimal versions for keyboard3 conformsTo?
```
2024-11-12T18:10:31.9474670Z [INFO] ---------------------< org.unicode.cldr:cldr-code >---------------------
2024-11-12T18:10:31.9475671Z [INFO] Building CLDR Code 46.1-SNAPSHOT
2024-11-12T18:10:31.9476689Z [INFO]   from pom.xml
2024-11-12T18:10:31.9477889Z [INFO] --------------------------------[ jar ]---------------------------------
2024-11-12T18:10:32.0279065Z [INFO] 
2024-11-12T18:10:32.0287810Z [INFO] --- exec-maven-plugin:3.1.0:java (default-cli) @ cldr-code ---
2024-11-12T18:10:32.1317386Z [WARNING] 
2024-11-12T18:10:32.1318757Z java.io.FileNotFoundException: Could not read schema /home/runner/work/cldr/cldr/keyboards/dtd/ldmlKeyboard.xsd
2024-11-12T18:10:32.1321075Z     at org.unicode.cldr.util.XMLValidator.main (XMLValidator.java:74)
2024-11-12T18:10:32.1322527Z     at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:279)
2024-11-12T18:10:32.1323685Z     at java.lang.Thread.run (Thread.java:829)
2024-11-12T18:10:32.1333930Z [INFO] ------------------------------------------------------------------------
2024-11-12T18:10:32.1334683Z [INFO] BUILD FAILURE
2024-11-12T18:10:32.1335463Z [INFO] ------------------------------------------------------------------------
2024-11-12T18:10:32.1348570Z [INFO] Total time:  0.596 s
2024-11-12T18:10:32.1350875Z [INFO] Finished at: 2024-11-12T18:10:32Z
2024-11-12T18:10:32.1352096Z [INFO] ------------------------------------------------------------------------
2024-11-12T18:10:32.1362092Z [ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.1.0:java (default-cli) on project cldr-code: An exception occurred while executing the Java class. Could not read schema /home/runner/work/cldr/cldr/keyboards/dtd/ldmlKeyboard.xsd -> [Help 1]
2024-11-12T18:10:32.1364826Z [ERROR] 
2024-11-12T18:10:32.1365835Z [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
2024-11-12T18:10:32.1367095Z [ERROR] Re-run Maven using the -X switch to enable full debug logging.
2024-11-12T18:10:32.1367880Z [ERROR] 
2024-11-12T18:10:32.1368797Z [ERROR] For more information about the errors and possible solutions, please read the following articles:
2024-11-12T18:10:32.1370340Z [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
2024-11-12T18:10:32.1577389Z Schema failed, rerun GenerateDtd
2024-11-12T18:10:32.1609745Z ##[group]Run mvn -s .github/workflows/mvn-settings.xml --file tools/pom.xml -pl cldr-code exec:java -Dexec.mainClass=org.unicode.cldr.util.XMLValidator -Dexec.cleanupDaemonThreads=false -Dexec.args="--schema=keyboards/dtd/ldmlKeyboardTest.xsd keyboards/test" || (echo 'Schema failed, rerun GenerateDtd' ; exit 0)
2024-11-12T18:10:32.1612476Z [36;1mmvn -s .github/workflows/mvn-settings.xml --file tools/pom.xml -pl cldr-code exec:java -Dexec.mainClass=org.unicode.cldr.util.XMLValidator -Dexec.cleanupDaemonThreads=false -Dexec.args="--schema=keyboards/dtd/ldmlKeyboardTest.xsd keyboards/test" || (echo 'Schema failed, rerun GenerateDtd' ; exit 0)[0m
2024-11-12T18:10:32.1670269Z shell: /usr/bin/bash -e {0}
2024-11-12T18:10:32.1670645Z env:
2024-11-12T18:10:32.1671031Z   JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/11.0.25-9/x64
2024-11-12T18:10:32.1671899Z   JAVA_HOME_11_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/11.0.25-9/x64
2024-11-12T18:10:32.1672511Z ##[endgroup]
```

Reply:
I ended up reverting the cldrVersion and conformsTo updates for ldmlKeyboard3 dtd/xsd, will see whether that fixes the problem

Reply:
Sounds like the right approach.